AgentDiscoveries-Backend/src/test/java/org/softwire/training/api/endToEnd/dummySeleniumIT.java

```diff
@@ -0,0 +1,26 @@
+package org.softwire.training.api.endToEnd;
+
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class dummySeleniumIT {
+    public static final String Target_Address=System.getProperty("target.address");
+
+    @Test
+    public void testTitle(){
+        WebDriver driver = new HtmlUnitDriver();
+        driver.get(Target_Address);
+        String title = driver.getTitle();
+        assertTrue(title.equals("Agent Discoveries"));
+    }
+
+    public void testOnline(){
```

Comment:
Does this need a `@Test` attribute?

Reply:
Ermm, apparently not as it was definitely running it anyway! But should definitely be there for clarity haha